changelog.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## NEXT
+
+- Removed `**kwargs` from the `add_node`, `add_edge`, and `add_edge_by_pair` methods of
```

Comment:
Suggest adding full path links to these methods so docs link nicely. Can just use the `CausalGraph` ones as that is the one you refer to first.